### ISSUE TITLE
`LineEndingConversion` in `.Rproj` is not properly respected in GUI

### ISSUE THREAD

**Issue by salim-b** [rstudio/rstudio#8169]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.963
    OS Version      : Ubuntu 20.04
    R Version       : 4.0.3

### Steps to reproduce the problem

1. Save the following lines to an empty file `test.Rproj` in an empty directory and double-click it in order to open RStudio.

    ```dcf
    Version: 1.0

    RestoreWorkspace: No
    SaveWorkspace: No
    AlwaysSaveHistory: Default

    EnableCodeIndexing: Yes
    UseSpacesForTab: Yes
    NumSpacesForTab: 2
    Encoding: UTF-8

    RnwWeave: Sweave
    LaTeX: pdfLaTeX

    AutoAppendNewline: Yes
    StripTrailingWhitespace: Yes
    LineEndingConversion: Posix

    QuitChildProcessesOnExit: Yes

    ```

2. Navigate to <kbd>Tools</kbd>→<kbd>Project Options…</kbd>→<kbd>Code Editing</kbd>. You should see that the config line `LineEndingConversion: Posix` wasn't parsed/respected by RStudio. The corresponding option in the GUI instead shows `(Use Default)`, i.e. looks like this (red underlined):

    ![Bildschirmfoto von 2020-10-23 01-11-52](https://user-images.githubusercontent.com/20040931/96943170-e00d9a00-14c6-11eb-8b45-4bfca5256827.png)

    While it actually should say **`Posix (LF)`**.

### Describe the problem in detail

This is especially irritating if you open a project on a system on which it hasn't been opened before and actually want to change a different Code Editing option (e.g. `Tab width`). If you do so and press <kbd>Ok</kbd>, the project's `.Rproj` file gets overwritten (fine) **and the line `LineEndingConversion: Posix` gets deleted (not fine)**.

### Describe the behavior you expected

I expected the project option `LineEndingConversion` in the `.Rproj` file to be properly respected in the RStudio GUI.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@salim-b Thank you for letting us know! It looks like changing the option within the UI doesn't stick:

1. Tools -> Project Options -> Code Editing
2. Set "Line ending conversion" to anything other than `(Use Default)`
3. Click OK
3. Tools -> Project Options -> Code Editing
    - The value is back to `(Use Default)`

I can reproduce this in RStudio Desktop 1.3.1093 and 1.4.953 on MacOS 11 beta 10, and in RStudio Server 1.4.953 on Red Hat 8.2.

**Comment by salim-b** [comment]
> It looks like changing the option within the UI doesn't stick (...)

Aah, you're absolutely right! Much easier to reproduce. 😆

**Comment by mikebessuille** [comment]
Another preference that wasn't moved over in 1.4

**Comment by kfeinauer** [comment]
Verified fixed 1.4.990
